dls-controls/malcolmjs
# Feature/subelement info #247

## Description

Added handling for info of a sub-element on a field (i.e. row in a table), in addition to adding an implementation of the info pane specifically for table rows (a couple of functions are still to be implemented).

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to http://localhost:3000/gui/PANDA/layout/SEQ1/table, add a row then click on its 'i' icon

## Agile board tracking

connect to #247 
connect to #243 
closes #247
